cmd/bootstrapper/configure_cni.go

```diff
@@ -57,19 +58,19 @@ func runConfigureCNICmd(cmd *cobra.Command, args []string) {
 	wmcb, err := bootstrapper.NewWinNodeBootstrapper(configureCNIOpts.installDir, "", "", configureCNIOpts.dir,
 		configureCNIOpts.config)
 	if err != nil {
-		log.Error(err, "could not create bootstrapper")
+		os.Stderr.WriteString(fmt.Sprint(err, "could not create bootstrapper"))
```

Comment:
Did you test what the output looks like? The formatting seems to be incorrect in all cases. Please post an example output.

Reply:
Updated